app/controllers/application_controller.rb

```diff
@@ -3,32 +3,33 @@ class ApplicationController < ActionController::API
 
     def encode_token(payload)
         JWT.encode(payload, 'my_secret')
-    end 
+    end
 
-    def auth_header 
-        request.headers["Authorization"]
-    end 
+    def auth_header
+        request.headers['Authorization']
+    end
 
     def decoded_token
-        if auth_header 
+        if auth_header
             token = auth_header.split(' ')[1]
             begin
                 JWT.decode(token, 'my_secret', true, algorithm: 'HS256')
```

Comment:
In this Line here we should hide the 'my_secrect' this is supposed to be a secret password. You can hide it with .env file